src/pymor/reductors/neural_network.py

```diff
@@ -106,6 +107,70 @@ def data_to_tensor(data):
             return StationaryNeuralNetworkModel(self.neural_network, error_estimator=None, **projected_operators)
 
 
+    class StationaryNeuralNetworkOutputReductor(BasicObject):
+
+        # TODO: allow for transformation/back_transformation of outputs (and inputs?)!
+        def __init__(self, fom, training_data, validation_data,
+                     hidden_layers='[P*3, P*3]', ann_mse=None, max_restarts=10, torch_seed=None, ann_train_params=None):
```

Comment:
Maybe `hidden_layers` could (by default) also depend on the number of outputs, as it was the case in the `StationaryNeuralNetworkReductor`? For instance something like
```suggestion
                     hidden_layers='[(O+P)*3, (O+P)*3]', ann_mse=None, max_restarts=10, torch_seed=None, ann_train_params=None):
```